core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -457,7 +457,7 @@ protected function getBackendModule($module, PickerInterface $picker = null)
 			$this->Template->main .= $response;
 
 			// Add the name of the parent element
-			if (isset($_GET['table']) && !empty($GLOBALS['TL_DCA'][$strTable]['config']['ptable']) && \in_array(Input::get('table'), $arrTables) && Input::get('table') != $arrTables[0])
+			if (isset($_GET['table']) && !empty($GLOBALS['TL_DCA'][$strTable]['config']['ptable']) && \in_array(Input::get('table'), $arrTables) && Input::get('table') != ($arrTables[0] ?? null))
```

Comment:
`$arrTables` is not related to label, so this change seems wrong.

Reply:
Yes, true. I skimmed through the file and fixed other parts as well. In this case `$arrTables` can be empty, see L339.